honeybee_schema/radiance/material.py

```diff
@@ -0,0 +1,280 @@
+"""Material Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+
+
+class Void(IDdBaseModel):
+    """Void modifier"""
+
+    type: constr(regex='^Void$') = 'Void'
```

Comment:
Update this to match the most updated to_dict once the RP is merged.

Reply:
From what I can see, this does match the honeybee-radiance PR. I think the only thing that is different is that it should be a lowercase `void` instead of a title case `Void`. The same goes for all of the modifiers here, which should have lowercase `'type'` keys.

Reply:
Fixed.